projects/packages/jetpack-mu-wpcom/src/features/domain-email-nag/domain-email-nag.php

```diff
@@ -49,6 +49,20 @@ function get_domain_with_unverified_email() {
 	return false;
 }
 
+/**
+ * Bump the MC stat for wpcom-domain-email-nag-shown.
+ *
+ * @param string $domain the domain to bump the stat for.
+ */
+function bump_mc_stats( $domain ) {
+	$pixel_url = sprintf(
+		'http://pixel.wp.com/g.gif?v=wpcom-no-pv&x_%s=%s',
+		'wpcom-domain-email-nag-shown',
+		$domain
+	);
+	wp_remote_get( $pixel_url );
```

Comment:
`wp_remote_get()` is blocking and will slow down the request. Could we display the pixel instead?

Reply:
I will switch to rendering an image instead. Thanks!